package ru.practicum.shareit.exceptions;

public record ErrorResponseDto(String errorMessage) {
}
